pkgpanda/cli.py

```diff
@@ -210,16 +210,16 @@ def main():
             # Run all checks
             sys.exit(run_checks(checks, install, repository))
     except ValidationError as ex:
```

Comment:
Yeah, at this level we should just have a single handler for expected exceptions and defer to their error messages. IMO the `Exception` handler should go away and unexpected exceptions should bubble up to the top, tracebacks and all.